### PR TITLE
#385

### PR DESCRIPTION
**Pull request by BrandonTsai**
#### Description

Declare REACT_APP_ENV environment variable when building docker image so that yarn build could only setup sentry and hotjar correctly.

#### QA URL

https://demo-roster.efcsydney.org/

#### Acceptance Criteria

- [x] Ensure that sentry and hotjar setting are correctly on QA environment.

### CODE REVIEW

File: docker/Dockerfile-prod [hunk 1]
```diff
@@ -1,8 +1,12 @@
 FROM mhart/alpine-node:8.10.0
 
+
+ARG REACT_APP_ENV=qa
+ENV REACT_APP_ENV=${REACT_APP_ENV}
+
```
Comment (liam-lai) [inline docker/Dockerfile-prod]: Do we need to put env at this file and docker-compose? I think we can put in compose file.

**Comment by josephj** [comment]
Solid! Hotjar works as expected.

https://insights.hotjar.com/r?site=724860&recording=1496123109&token=6a60034d7126e80f3c0af1109d188569

hotjar-qa

Sentry gets correct env now.

sentry-qa

File: docker/Dockerfile-prod (hunk 1 above)
Comment (BrandonTsai) [reply docker/Dockerfile-prod]: According to:
https://docs.docker.com/compose/compose-file/#environment

Note: If your service specifies a build option, `variables defined in environment are not automatically visible during the build`. Use the args sub-option of build to define build-time environment variables.

Comment (BrandonTsai) [reply docker/Dockerfile-prod]: And according to https://docs.docker.com/compose/compose-file/#args

First, we need to specify the arguments in Dockerfile, then  specify the arguments under the build key in docker-compose file.

Comment (liam-lai) [reply docker/Dockerfile-prod]: 👍

**Pull request merged**